templates/web/pages/apply.tmpl

```diff
@@ -219,34 +234,86 @@
 						<div class="field-body">
 							<div class="field">
 								<div class="control">
-									<input type="text" name="dietary-restrictions" class="input" value="{{ .FormRefill.DietaryRestrictions }}">
+									<input type="text" name="dietary-restrictions" class="input" maxlength="512" value="{{ .FormRefill.DietaryRestrictions }}">
 								</div>
 							</div>
 						</div>
 					</div>
 
-					<h3>Social</h3>
 					<div class="field is-horizontal">
 						<div class="field-label is-normal">
 							<label for="github" class="label">GitHub</label>
 						</div>
 						<div class="field-body">
 							<div class="field">
 								<div class="control">
-									<input type="text" name="github" class="input" value="{{ .FormRefill.Github }}">
+									<input type="text" name="github" class="input" maxlength="512" value="{{ .FormRefill.Github }}">
 								</div>
 							</div>
 						</div>
 					</div>
 
+					<div class="field">
+						{{ if .FormRefill.IsFirstHackathon }}
+						<input type="checkbox" name="is-first-hackathon" class="checkbox" checked>
+						{{ else }}
+						<input type="checkbox" name="is-first-hackathon" class="checkbox">
+						{{ end }}
+						<label for="is-first-hackathon" class="checkbox">This is my first hackathon</label>
+					</div>
+
 					<div class="field is-horizontal">
 						<div class="field-label is-normal">
-							<label for="linkedin" class="label">LinkedIn</label>
+							<label for="race" class="label">Where did you hear about BoilerMake?</label>
```

Comment:
Uhhhh race? Shouldn't this be referrer?

Reply:
And that's why we do code reviews.  Good catch